--- proxydump/20111217-000715.680-fmshttpstg.bbc.co.uk.edgesuite-staging.net--hds-live-streams-livepkgr-streams-_definst_-inlet2-inlet2.dump	2011-12-17 11:17:57.463408547 +0000
+++ proxydump/20111217-000720.563-fmshttpstg.bbc.co.uk.edgesuite-staging.net--hds-live-streams-livepkgr-streams-_definst_-inlet2-inlet2.dump	2011-12-17 11:17:57.479408768 +0000
@@ -4,7 +4,7 @@
 	is live: true
 	movie id: 
 	timescale: 1000
-	current media time: 68830332
+	current media time: 68838319
 	smpte timecode offset: 0
 	servers:
 	qualities:
@@ -194,7 +194,7 @@
 		segment = 1718, fragments = 10
 		segment = 1719, fragments = 10
 		segment = 1720, fragments = 10
-		segment = 1721, fragments = 7
+		segment = 1721, fragments = 9
 fragments:
 fragment run table:
 	is update: false
